doc/releases/release-notes-3.1.rst

```diff
@@ -1123,3 +1123,799 @@ Issue Related Items
 
 These GitHub issues were addressed since the previous 3.0.0 tagged
 release:
+
+* :github:`46241` - Bluetooth: Controller: ISO: Setting CONFIG_BT_CTLR_ISO_TX_BUFFERS=4 breaks non-ISO data
+* :github:`46223` - Release notes for v3.1.0: USB
+* :github:`46222` - Release notes for v3.1.0: timer
+* :github:`46220` - Release notes for v3.1.0: documentation
+* :github:`46219` - Release notes for v3.1.0: DT
+* :github:`46218` - Release notes for v3.1.0: SPI
+* :github:`46217` - Release notes for v3.1.0: I2S
+* :github:`46216` - Release notes for v3.1.0: I2C
+* :github:`46215` - Release notes for v3.1.0: GPIO
+* :github:`46214` - Release notes for v3.1.0: ethernet
+* :github:`46213` - Release notes for v3.1.0: eeprom
+* :github:`46212` - Release notes for v3.1.0: dma
+* :github:`46211` - Release notes for v3.1.0: disk
+* :github:`46210` - Release notes for v3.1.0: arm64
+* :github:`46209` - Release notes for v3.1.0: counter
```

Comment:
do these release note piece need to be in the release notes issue list?  Seems like it's not really of interest for an end user to see such administravia.